### DISCUSSION TITLE
Raise your hand button -- without a video call

### DISCUSSION THREAD

**Discussion by mattdm** [element-hq/element-meta#2351] [Ideas]
Today's common videoconferencing platforms have a "raise your hand" feature, which keeps a queue of people waiting to talk. There is a request for this [here, for element-call](https://github.com/element-hq/element-call/issues/130). However, I would love for this to be more general.

In Fedora, from long tradition back to IRC, we hold many meetings in text.  We have a bot to help with this, and it's slightly formal.... you can see past meeting logs at https://meetbot.fedoraproject.org/.

For big meetings in the past in IRC, we sometimes used a convention of typing `!` to ask to speak, and `?` to note that you have a question. We could do something like this (possibly including support in our meeting-logging bot), but that seems kind of messy.

Instead, I would like to request that the solution for https://github.com/element-hq/element-call/issues/130 be something general which could work in all rooms, whether or not a call is active.

**Comment by toger5** [comment]
I really like the general approach you propose here. Not making it a custom thing but reusing what is there and transporting it into a call.

This would allow us to not spec new things and clients not supporting specific call features can easily still see what is happening.
Another example I imagine could be to display any `/me` message as a bubble ontop/next/near the video tile of a participant. This would already allow a lot of workarounds. It can be done as custom rendering but can also be consumed by looking at the timeline while beeing part of a call.
